app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinPresenter.kt

```diff
@@ -70,7 +70,6 @@ class NewCreatePinPresenter(
 
     private fun checkBiometricAvailability() {
         if (authInteractor.getBiometricStatus() < BiometricStatus.AVAILABLE) {
-            createWalletInteractor.finishAuthFlow()
```

Comment:
@kuznetsov fyi

Reply:
It's not okay :(

Reply:
We have to finish auth  flow if user has no biometrics and pin is enough to start main screen. But now looks like instead of mock web3auth we can't generate user and finishAuthFlow() is crashing

Reply:
i revert this